### PR TITLE
Blog Post: 18F checks in with Emily Read and the USGS Water Resources Mission Area Projects

### PR DESCRIPTION
**Pull request by amymok**
# Pull request summary
New blog post.  https://github.com/18F/blog-drafts/issues/830

(Optional) Closes #(issue_number).

## Reminder - please do the following before assigning reviewer

- [ ] update readme
  - Blog post only, no new code changes, so no update to readme
- [ ] For frontend changes, ensure design review
  - No front end changes

And make sure that automated checks are ok

- fix houndci feedback
- ensure tests pass
- federalist builds
- no new SNYK vulnerabilities are introdcued

### CODE REVIEW

**Comment by alexsobledotgov** [comment]
@amymok Since you tagged me as a reviewer on this PR, I gave the preview build a look and will leave a few minor, non-blocking suggestions.

**Comment by alexsobledotgov** [comment]
For ease of review, here's a [shortcut link to blog post preview build](https://federalist-c58f58dc-a215-4616-a54c-12b5eb011096.sites.pages.cloud.gov/preview/18f/18f.gsa.gov/aymok/usgs-wma-check-in-blog/2023/02/23/18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects/).

**Review by alexsobledotgov — APPROVED** [review]
Changing status to "approved" because all my suggestions here are non-blocking!

File: _posts/2023-02-23-18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+_This is the second in a series of blog posts where 18F checks in with our partners throughout the government. Previously in this series: [18F Checks In With Jerome Lee and the eAPD Project](https://18f.gsa.gov/2022/11/29/18f-checks-in-with-jerome-lee-and-the-eapd-project/)._
+
+![USGS hydrologic technician Travis Gibson confirms Great Salt Lake water levels at the SaltAire gauge.]({{ site.baseurl }}/assets/blog/usgs-wma-checkin/20210722_150403.jpg)
+USGS hydrologic technician Travis Gibson confirms Great Salt Lake water levels at the SaltAire gauge. Source: [USGS Multimedia Gallery](https://www.usgs.gov/media/images/great-salt-lake-reaches-new-historic-low)
+{: .font-alt-xs }
+
+18F partnered with the U.S. Geological Survey (USGS) on a few different projects from March 2020 to June 2022. We worked on a wide variety of projects together, including: [Water Resources Mission Area (WMA) Platform Modernization Path Analysis](https://18f.gsa.gov/2020/08/06/doing-user-research-to-design-the-next-gen-wdfn/), Experiment & Iterate, and Acquisition Consulting; National Groundwater Monitoring Network (NGWMN) Path Analysis; and National Water Information System (NWIS) Data Delivery Strategy.
```
[the remark this answers is not in the document]
Comment (amymok) [reply _posts/2023-02-23-18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects.md]: Removed unused acronyms

File: _posts/2023-02-23-18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  - u.s. geological survey
+excerpt: >
+  At 18F, we like to keep in touch with our project partners. 18F partnered with the USGS Water Resources Mission Area (WMA) from March 2020 to June 2022 on a wide variety of projects together. We caught up with Emily Read, Chief for the Web Communications Branch at USGS Water Resources Mission Area.
+---
+
+_This is the second in a series of blog posts where 18F checks in with our partners throughout the government. Previously in this series: [18F Checks In With Jerome Lee and the eAPD Project](https://18f.gsa.gov/2022/11/29/18f-checks-in-with-jerome-lee-and-the-eapd-project/)._
+
+![USGS hydrologic technician Travis Gibson confirms Great Salt Lake water levels at the SaltAire gauge.]({{ site.baseurl }}/assets/blog/usgs-wma-checkin/20210722_150403.jpg)
+USGS hydrologic technician Travis Gibson confirms Great Salt Lake water levels at the SaltAire gauge. Source: [USGS Multimedia Gallery](https://www.usgs.gov/media/images/great-salt-lake-reaches-new-historic-low)
+{: .font-alt-xs }
```
[the remark this answers is not in the document]
Comment (amymok) [reply _posts/2023-02-23-18f-checks-in-with-emily-read-and-the-usgs-water-resources-mission-area-projects.md]: I am leaving this as-is.  As I was browsing through other blog posts, if the images is not part of the text, mostly they are at the top.  So I will leave it there.

**Pull request merged**